### PR TITLE
Change path in configuring hostname verification doc

### PR DESCRIPTION
**Pull request by ruthryi**
Fixes
- https://github.com/wso2-enterprise/wso2-apim-internal/issues/2852

### CODE REVIEW

File: en/docs/install-and-setup/setup/security/enabling-hostname-verification.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The hostname verification is **disabled** by default. This is done using the `o
 
 !!! note "Strict hostname verification"
 
-    When applying the strict hostname verification, you need to change the configuration in the `site/public/conf/settings.js` file of the Publisher and Admin Portal and the `site/public/theme/setting.js` file of the Developer Portal.
+    When applying the strict hostname verification, you need to change the configuration in the `<APIM_HOME>/repository/deployment/server/webapps/publisher/site/public/conf/settings.json` file of the Publisher and Admin Portal and the `site/public/theme/setting.js` file of the Developer Portal.
```
Comment (tharikaGitHub) [inline en/docs/install-and-setup/setup/security/enabling-hostname-verification.md:21]: Let's add the relevant full path to devportal settings file as well.

Comment (ruthryi) [reply en/docs/install-and-setup/setup/security/enabling-hostname-verification.md]: Addressed via https://github.com/wso2/docs-apim/pull/7055/commits/1a5991c978624e7b60f1941706cbf0362bde6f59

Comment (tharikaGitHub) [reply en/docs/install-and-setup/setup/security/enabling-hostname-verification.md]: As this is referring to the path in the Devportal app, it has to be `<APIM_HOME>/repository/deployment/server/webapps/devportal/site/public/theme/settings.json`. Shall we correct it Ruthryi?

Comment (ruthryi) [reply en/docs/install-and-setup/setup/security/enabling-hostname-verification.md]: Corrected with https://github.com/wso2/docs-apim/pull/7055/commits/6a21267d961d6df499fbb48e70b0c994a0de345d

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**